articles/proofs-arguments-and-zero-knowledge/chapter_12.md

```diff
@@ -0,0 +1,276 @@
+---
+Title: "Σ-Protocols and Commitments from Hardness of Discrete Logarithm"
+Status: Done
+Level: "3"
+---
+
+# Chapter 12: $Σ$-Protocols and Commitments from Hardness of Discrete Logarithm
+
+## 12.1 Cryptographic Background
+
+### 12.1.1 A Brief Introduction to Groups
+
+This is the definition of [group](../../terms/group.md).
+
+### 12.1.2 The Discrete Logarithm Problem and Background on Elliptic Curves
+
+#### 12.1.2.1 Discrete Log Problem
+
+For a specified group $G$ the discrete logarithm problem takes as input two group elements $g$ and $h$, and the goal is to output a
+positive integer $i$ such that $g^i = h$ (if $G$ is of prime order then such an $i$ is guaranteed to exist). The discrete logarithm
+problem is believed to be computationally intractable in certain groups $G$. An important caveat is that quantum computers can solve
+the discrete logarithm problem in polynomial time via Shor’s algorithm. Hence, cryptosystems whose security is based on the assumed
+hardness of the discrete logarithm problem are not post-quantum secure.
+
+#### 12.1.2.2 Elliptic Curve Groups
```

Comment:
Move this to `elliptic_curve_group.md` and link it with this chapter and the elliptic_curve definition